src/secretsManager.ts

```diff
@@ -0,0 +1,36 @@
+import AWS from 'aws-sdk';
+
+const { IS_OFFLINE, AWS_REGION, STAGE } = process.env;
+export default class SecretsManager {
+    static initialize() {
+        if (IS_OFFLINE === 'true') {
+            AWS.config.update({
+                region: 'us-west-2',
+                accessKeyId: process.env.ACCESS_KEY,
+                secretAccessKey: process.env.SECRET_KEY,
+            });
+        }
+    }
+
+    static async getIntegrationTransformUrl(): Promise<string> {
+        const ssm = new AWS.SSM();
+        let path = process.env.INTEGRATION_TRANSFORM_PATH ?? 'fake_path';
+        if (IS_OFFLINE === 'true') {
+            path = `fhir-service.integration-transform.${AWS_REGION}.${STAGE}.url`;
+        }
+        const data = await ssm
+            .getParameter({
+                Name: path,
+                WithDecryption: true,
+            })
+            .promise();
+
+        if (data.Parameter?.Value === undefined) {
+            console.error(`Unable to get INTEGRATION_TRANSFORM_PATH. Path: ${path}`);
+        }
+
+        return data.Parameter?.Value ?? '';
+    }
+}
+
+SecretsManager.initialize();
```

Comment:
It's weird to have to call a static class method before exporting. Do consumers of this class have any reason to call `initialize `?

I think that this file should simply export a function like `getIntegrationTransformUrl `. I don't think a class makes sense here if it's only going to have public static methods.

Reply:
In this `package`, we only make calls to AWS in this class. Therefore, `initialize` only needs to be run when this class is being used. 

I wanted to encapsulate the logic of `initialize` and making requests to System Manager param store in one place. The alternative would be to put the `initialize` code in `config.ts` also and export `getIntegrationTransformUrl`. However, none of the other code in `config` needs to make calls to AWS.

Reply:
Updated code to have this file export functions while still keeping updates of AWS config in this file.